### DISCUSSION TITLE
Why...Unknown manufacturer, 0xXXXX, Unknown product 0xXXXX

### DISCUSSION THREAD

**Discussion by JasFos** [zwave-js/zwave-js#4025] [Q&A]
Why do my devices keep going to "Unknown manufacturer, 0xXXXX, Unknown product 0xXXXX" ?
This is the third time it's happened, very annoying.  Everything in HA fails when this happens.

I end up having to remove them and add them again to get them to work.

zWave stick info HUSBZB-1 (Silicon Labs Z-Wave 500 Series Controller ACC-UZB3)

![zwaveunknown](https://user-images.githubusercontent.com/15234321/147978718-ad2c14d6-8b42-44db-8565-fc8bdc5947b5.jpg)

**Comment by AlCalzone** [comment]
Can't say without driver logs. Please open an issue and provide the requested information.

**Reply by JasFos** [reply]
Which driver logs and what's the easy way to get them?

**Reply by AlCalzone** [reply]
The issue template tells you, just open one:
![grafik](https://user-images.githubusercontent.com/17641229/147987190-7844d3e0-06b5-43cf-87e5-ee935bb68047.png)

**Reply by JasFos** [reply]
Submitted #4030
thanks
